src/uct/sm/base/sm_iface.c

```diff
@@ -88,11 +115,36 @@ UCS_CLASS_INIT_FUNC(uct_sm_iface_t, uct_iface_ops_t *ops, uct_md_h md,
 
     self->config.bandwidth = sm_config->bandwidth;
 
+    self->shmid = shmget(IPC_PRIVATE, sizeof(uint64_t),
```

Comment:
so if SysV is not supported, posix will not work as well?

Reply:
right, but without sysV - most of unix systems are not operable
as I remember sysV supported by all linux & freeBSD/MacOS systems

Reply:
if we already have support for posix without sysv dependency - let's keep it that way

Reply:
pls use the transport itself and its mappers to test reachability, not direct calls to sysv